src/styles/_header.scss

```diff
@@ -31,21 +31,19 @@
   justify-content: center;
   height: 4.25rem;
 
-  &__text {
-    display: table-cell;
-    vertical-align: middle;
-  }
-
   &__item {
     margin: 0 1.875rem;
-    font-size: 0.75rem;
-    display: table;
+    font-size: 1rem;
+    display: flex;
+    align-items: center;
     .contacts-icon {
```

Comment:
Does `contacts-icon` is a separate block? Or it's an inner element of photo block?